src/main.ts

```diff
@@ -154,13 +154,9 @@ export default class LlmShortcutPlugin extends Plugin {
 
       logger.debug(`Added command ${readableCommandName}`);
 
-      const { content, options } = await this.parseCommandPromptFromFile(file);
-
-      this.addCommandBasedOnPrompt({
+      this.addCommandBasedOnFile({
         name: readableCommandName,
-        path: file.path,
-        prompt: content,
-        options,
+        promptFilePath: file.path,
```

Comment:
Only keep the prompt file path in the command (in case of changes, we only need to delete/add/update the file path)